creator-node/src/services/stateMachineManager/stateMonitoring/findReplicaSetUpdates.jobProcessor.js

```diff
@@ -109,6 +117,7 @@ module.exports = async function ({
   }
 
   return {
+    cNodeEndpointToSpIdMap: CNodeToSpIdMapManager.getCNodeEndpointToSpIdMap(),
```

Comment:
still need this with the other PR?

Reply:
not 100% because we could cross reference the other queue from the other PR and find out what the mapping was at the time the update-replica-set job ran. however, this makes it easier to see all the info in this one job especially and it's not a terribly large object to store. are you ok with keeping it for that reason?

Reply:
yeah